node/libsignal_client.d.ts

```diff
@@ -66,11 +68,13 @@ export function SenderCertificate_GetServerCertificate(cert: SenderCertificate):
 export function SenderCertificate_GetSignature(obj: SenderCertificate): Buffer;
 export function SenderCertificate_New(senderUuid: string, senderE164: string | null, senderDeviceId: number, senderKey: PublicKey, expiration: number, signerCert: ServerCertificate, signerKey: PrivateKey): SenderCertificate;
 export function SenderCertificate_Validate(cert: SenderCertificate, key: PublicKey, time: number): boolean;
+export function SenderKeyDistributionMessage_Create(name: SenderKeyName, store: unknown): Promise<SenderKeyDistributionMessage>;
```

Comment:
To avoid this `unknown`, maybe the abstract class should live in libsignal_client.d.ts.in?

Reply:
Abstract class has interfaces which refer to the wrapper types, so that doesn't work. What I did instead is split it into an interface with the functions that we invoke in Rust, then the abstract class in index.ts implements that interface in terms of the functions on the abstract class that are operating on the wrapper types.